docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -289,6 +289,8 @@ set name for unique queues. Individual queues will default to
 
 - `INSTALL_LOCK`: **false**: Disallow access to the install page.
 - `SECRET_KEY`: **\<random at every install\>**: Global secret key. This should be changed.
+- `WEB_PUSH_PUBLIC_KEY`: **\<random at every install\>**: VAPID key pair used for Web Push notifications
+- `WEB_PUSH_PRIVATE_KEY`: **\<random at every install\>**: VAPID key pair used for Web Push notifications
```

Comment:
`PUSH_NOTIFICATIONS_PUBLIC_KEY` and `PUSH_NOTIFICATIONS_PRIVATE_KEY` would be more better imho.

Reply:
In the codebase I've tried to be consistent with 'Web Push', but I could change the user-facing config quite easily